### DISCUSSION TITLE
how the lammps get the traj file with four different potential.

### DISCUSSION THREAD

**Discussion by maoxinxina** [deepmodeling/dpgen#1301] [Q&A] [answered]
the submit file of lammps is :pair_style      deepmd ../graph.000.pb ../graph.001.pb ../graph.002.pb ../graph.003.pb out_freq 20 out_file model_devi.out , I want to know how the lammps get the traj file, with the first potential or a weight of the four potential.

**Comment by njzjz** [comment, marked as answer]
Please read the documentation: https://docs.deepmodeling.com/projects/deepmd/en/master/third-party/lammps-command.html#pair-style-deepmd

**Comment by maoxinxina** [comment]
Thanks。
